tools/pep_pointer/pep_pointer.xml

```diff
@@ -1,7 +1,7 @@
-<tool id="pep_pointer" name="PepPointer" version="0.1.3">
+<tool id="pep_pointer" name="PepPointer" version="0.1.3+galaxy1">
```

Comment:
I guess here we can just increase the version to 0.1.4, because it does not depend on external software ..